data/json/recipes/recipe_weapon.json

```diff
@@ -195,6 +195,31 @@
       [ [ "duct_tape", 20 ], [ "filament", 20, "LIST" ] ]
     ]
   },
+  {
+    "type": "recipe",
+    "result": "spear_stone",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_PIERCING",
+    "skill_used": "fabrication",
+    "skills_required": [ "survival", 1 ],
+    "difficulty": 2,
+    "time": 2400,
+    "autolearn": true,
+    "qualities": [ { "id": "CUT", "level": 1 }, { "id": "HAMMER", "level": 1 } ],
+    "components": [
+      [ [ "stick_long", 1 ], [ "pool_cue", 1 ] ],
+      [ [ "rag", 1 ], [ "felt_patch", 1 ], [ "leather", 1 ], [ "fur", 1 ] ],
+      [
+        [ "plant_fibre", 20 ],
+        [ "sinew", 20 ],
+        [ "thread", 20 ],
+        [ "yarn", 20 ],
+        [ "duct_tape", 20 ],
+        [ "cordage_short", 1, "LIST" ]
+      ],
+      [ [ "rock", 1 ], [ "ceramic_shard", 1 ], [ "sharp_rock", 1 ] ]
```

Comment:
This is the real problem, and unless you want to get into the wild world of knapping, it's going to block this PR.

You can't just tie a rock to a stick and call it a spear, but that's exactly what this recipe is doing.  The reason we don't have stone tools in mainline is we don't have a knapping system.  

MST just ignored the need for one and made stone tools magically appear as needed despite knapping being a time-consuming and skill-intensive technique.